ui/app/styles/core/element-styling.scss

```diff
@@ -0,0 +1,163 @@
+/* These come from bulma */
```

Comment:
NIT: Probably safe to remove the old class selectors e.g. -webkit-font-smoothing: auto; Will need to double check.